app/default/.elasticsearch-http-requests/init.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -eu
+
+SCRIPTPATH="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )";
+
+"$SCRIPTPATH/request.sh" "$SCRIPTPATH/Index/Delete.http" || true
+"$SCRIPTPATH/request.sh" "$SCRIPTPATH/Index/Create.http" && \
+"$SCRIPTPATH/request.sh" "$SCRIPTPATH/Index/Bulk.http" && sleep 1 && \
+"$SCRIPTPATH/request.sh" "$SCRIPTPATH/Index/Count.http"
+
+printf "Done!\n"
```

Comment:
@212223 worth adding new line at the end of this file and others 😉

Reply:
Done in [718b5aa](https://github.com/lrynek/painless-car-rental/pull/1/commits/718b5aaedab0d30b1611f0d20f60840eb2b24cb8)